File: cs2ts/TestClass.ts
namespace cs2ts
{
    export  class TestClass
    {
        public FieldA: string;
        public FieldB: string;
        public PropertyC: number;
        constructor(a: string, b: number)
        {
            this.FieldA = a;
            this.FieldB = b.ToString();
            this.PropertyC = 12.3;
        }
        public static Test1(): void
        {
            System.Console.WriteLine("Test1");
        }
        public Test2(): void
        {
            System.Console.WriteLine("Test2");
        }
    }
}

File: cs2ts/TestClass.cs
namespace cs2ts
{
    public class TestClass
    {

        public string FieldA;
        public string FieldB;

        public float PropertyC { get; set; }
        

        public TestClass(string a, double b)
        {
            FieldA = a;
            FieldB = b.ToString();
            PropertyC = 12.3f;
        }

        public static void Test1()
        {
            System.Console.WriteLine("Test1");
        }


        public void Test2()
        {
            System.Console.WriteLine("Test2");
        }

    }
}